lib/screen/feed.dart

```diff
@@ -0,0 +1,257 @@
+import 'dart:convert';
+
+import 'package:flutter/material.dart';
+import 'package:google_fonts/google_fonts.dart';
+import 'package:sochem/models/post_model.dart';
+import 'package:cached_network_image/cached_network_image.dart';
+import 'package:intl/intl.dart';
+import 'package:http/http.dart' as http;
+
+import '../utils/constants.dart';
+
+class FeedScreen extends StatefulWidget {
+  @override
+  _FeedScreenState createState() => _FeedScreenState();
+}
+
+class _FeedScreenState extends State<FeedScreen> {
+  //Posts ViewModel
+  List<Post> _posts = [];
+  Future<List<Post>> fetchPosts() async {
+    var response =
+        await http.get(Uri.parse('https://api.npoint.io/bf243fd49e6b8783f02b'));
+
+    List<Post> posts = [];
+    if (response.statusCode == 200) {
+      var postsJson = json.decode(response.body);
+      for (var postJson in postsJson) {
+        posts.add(Post.fromJson(postJson));
+      }
+    }
+    return posts;
+  }
+
+  //Posts View
+  @override
+  void initState() {
+    fetchPosts().then((value) {
+      setState(() {
+        _posts.addAll(value);
+      });
+    });
+    super.initState();
+  }
+
+  Widget _buildPost(int index) {
+    // var postCreatedAt = DateTime(
```

Comment:
These changes show the time when the event is shared or posted. It can be implemented once backend requirements are met. Right now, we have the hard-coded time of posts.